### ISSUE TITLE
non-consistent dump in ibmvpc test TestAllWithComparison/iks_workers_large#05 (commented)

### ISSUE THREAD

**Issue by ShiriMoran** [np-guard/vpc-network-config-analyzer#847]
The dump seems in drawio area and is not consistent
https://github.com/np-guard/vpc-network-config-analyzer/actions/runs/10668928052/job/29569713712?pr=843
```

panic: runtime error: integer divide by zero [recovered]
	panic: runtime error: integer divide by zero

goroutine 85 [running]:
testing.tRunner.func1.2({0xa2f540, 0x1020dc0})
	/opt/hostedtoolcache/go/1.22.4/x64/src/testing/testing.go:1631 +0x24a
testing.tRunner.func1()
	/opt/hostedtoolcache/go/1.22.4/x64/src/testing/testing.go:1634 +0x377
panic({0xa2f540?, 0x1020dc0?})
	/opt/hostedtoolcache/go/1.22.4/x64/src/runtime/panic.go:770 +0x132
github.com/np-guard/vpc-network-config-analyzer/pkg/drawio.(*layoutOverlap).potentialBypassPoints(0xc0148abd10, {0x60f7be?, 0x60539b?}, {0xacaf20?, 0xc00003f750?}, {0x623cad?, 0xc03e6469c0?})
	/home/runner/work/vpc-network-config-analyzer/vpc-network-config-analyzer/pkg/drawio/layoutOverlap.go:195 +0x554
github.com/np-guard/vpc-network-config-analyzer/pkg/drawio.(*layoutOverlap).getBypassPoint(0xc0148abd10, {0xc03e6469c0?, 0x0?}, {0x4d?, 0x3e00618800020403?}, {0xc110e0?, 0xc03e63cb40?}, {0xc110e0, 0xc03e63cb40})
	/home/runner/work/vpc-network-config-analyzer/vpc-network-config-analyzer/pkg/drawio/layoutOverlap.go:216 +0x4b
github.com/np-guard/vpc-network-config-analyzer/pkg/drawio.(*layoutOverlap).handleLinesOverIcons(0xc0148abd10)
	/home/runner/work/vpc-network-config-analyzer/vpc-network-config-analyzer/pkg/drawio/layoutOverlap.go:166 +0x36d
github.com/np-guard/vpc-network-config-analyzer/pkg/drawio.(*layoutOverlap).fixOverlapping(0xc0148abd10)
	/home/runner/work/vpc-network-config-analyzer/vpc-network-config-analyzer/pkg/drawio/layoutOverlap.go:72 +0x39
github.com/np-guard/vpc-network-config-analyzer/pkg/drawio.(*layoutS).layout(0xc00003faa0)
	/home/runner/work/vpc-network-config-analyzer/vpc-network-config-analyzer/pkg/drawio/layout.go:103 +0x6db
```

**Comment by ShiriMoran** [comment]
also here https://github.com/np-guard/vpc-network-config-analyzer/actions/runs/10697602099/job/29655141446?pr=843

**Comment by zivnevo** [comment]
@haim-kermany can this issue be closed too?

**Comment by zivnevo** [comment]
Fixed by #900
